### ISSUE TITLE
bruteforce_re generates incorrect values

### ISSUE THREAD

**Issue by tlansec** [dhondta/python-tinyscript#29]
```python
from tinyscript import * 

for e in ts.bruteforce_re('[A-Z]{5}[0-9]{2}'):
    print(e)
```

Produces:

```text
UU44
UU46
UU45
...
```

**Comment by dhondta** [comment]
Hi @tlansec 
Thank you for reporting.
It seems that I made a mistake in the [nested function](https://github.com/dhondta/python-tinyscript/blob/310031505b1c9a5b000d6c31e639ae661a9d5843/src/tinyscript/preimports/regex.py#L145) of the big recursive function in the `regex.py` module ; the `start` and `end` bounds were changed when moving from the first (`[A-Z]{5}`) to the second group (`[0-9]{2}`) of your regular expression, hence keeping `2` as the new ènd` when executing the nested function even for the first group.
I'll fix this in my next commit by sending the `start` and `end` from the outer scope as parameters of the nested function.
